source/daplink/usb2uart/usbd_user_cdc_acm.c

```diff
@@ -85,7 +85,11 @@ int32_t USBD_CDC_ACM_PortSetLineCoding(CDC_LINE_CODING *line_coding)
     UART_Config.DataBits    = (UART_DataBits) line_coding->bDataBits;
     UART_Config.Parity      = (UART_Parity)   line_coding->bParityType;
     UART_Config.StopBits    = (UART_StopBits) line_coding->bCharFormat;
+#if defined(BOARD_VBLUNO51) || defined(BOARD_VBLUNO52)
+    UART_Config.FlowControl = UART_FLOW_CONTROL_RTS_CTS;
+#else
     UART_Config.FlowControl = UART_FLOW_CONTROL_NONE;
+#endif    
```

Comment:
Flow control could be useful to other devices as well. Could you add a function to enable it like the sam3u2c has:
https://github.com/mbedmicro/DAPLink/blob/master/source/hic_hal/atmel/sam3u2c/uart.c#L397

Then you can use prerun_board_config() to enable it:
https://github.com/mbedmicro/DAPLink/blob/master/source/board/ncs36510rf.c#L21